client/src/client.ts

```diff
@@ -25,6 +25,11 @@ export async function activate(context: ExtensionContext) {
         path.join('erlang_ls', '_build', 'default', 'bin', 'erlang_ls')
     );
 
+    let executablePath = workspace.getConfiguration('erlang_ls').executable_path;
```

Comment:
I had a local branch started for this but my Googling failed me when it came to finding how to access extension configurations.

Reply:
I copied and pasted from the haskell one, where the feature was added by a kind contributor.

The interesting thing is that the config json blob goes to the language server too when it changes.

Reply:
Why adding an extra variable here? Wouldn't it be nicer to just have `serverPath`, to specify the `.../_build/default/bin/erlang_ls` path as the default one in the config and allow the user to overwrite that one?

Reply:
@robertoaloi That's a good point, I would only hesitate on the directory separator (`/` or `\`) to make sure this works for Windows. Although you could argue that the path could then be fixed by a user if it didn't, I like when things just work out of the box.